### ISSUE TITLE
Windows build: missing dll

### ISSUE THREAD

**Issue by jeromecc** [FreeMedForms/freemedforms#71]
Qt5 Windows build is not managed correctly and it leads to missing dll files:

printsupport\windowsprintersupport.dll
libwinpthread-1.dll
libstdc++-6.dll
Qt5Core.dll
icuin52.dll
icuuc52.dll
icudt52.dll
Qt5Widgets.dll
Qt5Gui.dll
Qt5Network.dll
Qt5PrintSupport.dll
Qt5Script.dll
Qt5Sql.dll
Qt5Svg.dll
Qt5Xml.dll
platforms/qwindows.dll
platforms/qminimal.dll
platforms/qoffscreen.dll
msvcr120.dll
...

We need to update buildspecs/install.pri and scripts/win/win_release.bat to take into account the new Qt5 default.

About CRT & quazip/zlib

Microsoft changed its policy regarding c runtime library shipping with Windows8: https://msdn.microsoft.com/en-us/library/windows/hardware/dn448963%28v=vs.85%29.aspx
tldr: We need to ship msvcr120.dll in the bundle or "Use the Visual C++ Redistributable Package (VCRedist_x86.exe, VCRedist_x64.exe, VCRedist_arm.exe), which deploys the CRT under System32. In this case, the redistributable package can be updated automatically".

The "missing msvcr120.dll" message happens only on Windows 8. From reports of users, it doesn't prevent the application to start and run normally...

**Comment by jeromecc** [comment]
hotfix-0.9.6-i71 merged
